app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -19,7 +20,14 @@ def raw_stop
 
   def raw_reboot_guest
     with_provider_connection(:service => 'PowerIaas') do |power_iaas|
-      power_iaas.reboot_pvm_instance(ems_ref)
+      power_iaas.soft_reboot_pvm_instance(ems_ref)
+    end
+    update!(:raw_power_state => "off")
```

Comment:
For `raw_power_state` I think the options for power are `ACTIVE`/`INACTIVE` according to https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb#L33-L40
(I'm guessing on the `INACTIVE` @jaywcarman keep me honest here)

Reply:
@agrare @Kuldip-Nanda  it's actually `ACTIVE` / `SHUTOFF` :shrug:

Reply:
thanks, just missed it.

Reply:
For the record, the power states in PowerVS come from OpenStack:
https://docs.openstack.org/nova/latest/reference/vm-states.html

MIQ OpenStack provider power states.
https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/app/models/manageiq/providers/openstack/cloud_manager/vm/operations/power.rb